app/models/miq_worker.rb

```diff
@@ -19,7 +19,7 @@ class MiqWorker < ApplicationRecord
   scope :with_miq_server_id, ->(server_id) { where(:miq_server_id => server_id) }
   scope :with_status,        ->(status)    { where(:status => status) }
 
-  cattr_accessor :my_guid, :instance_accessor => false
+  class_attribute :my_guid, :instance_accessor => false
```

Comment:
We always set this value on MiqWorker [here](https://github.com/kbrock/manageiq/blob/hidden_attributes/app/models/miq_worker/runner.rb#L131) - think that is the only spot. But of interest is the `MiqWorker` mention.


It looks like the place we read the value is in this file [here](https://github.com/kbrock/manageiq/blob/hidden_attributes/app/models/miq_worker.rb#L267) - that does not have a qualifier, so I worry that it will read out of the subclass and get the wrong value.

This also looks like it has possible bleed over to MiqServer -- which is not good.  Do note that over there, they access via `@@` so changing the cattr to class_attribute deserves more diligence over there

Please change the read to fully qualified read and then this looks good to me